### ISSUE TITLE
YARA External Variables - FromRaccine > ParentFromRaccine

### ISSUE THREAD

**Issue by Neo23x0** [Neo23x0/Raccine#74]
The second value shouldn't be there, right? 

![image](https://user-images.githubusercontent.com/2851492/97219533-6e856280-17ca-11eb-95d1-dd844a0199e7.png)

**Comment by JohnLaTwC** [comment]
Correct, it's there because the defines for the child and the parent are combined together and each emits this define so they get concatenated together.

**Comment by JohnLaTwC** [comment]
I removed these defines all together so we should be able to close this issue.
